modules/vue-router/concepts/GlobalBeforeEach.js

```diff
@@ -0,0 +1,17 @@
+//@ts-check
+
+import { GeneratingConcept } from '@storefront-x/core'
+
+export default class GlobalBeforeEach extends GeneratingConcept {
```

Comment:
Rename to VueRouterBeforeEach + rename file + write at least one Playwright test

Reply:
done